css/style.css

```diff
@@ -1,3 +1,522 @@
+  border: 3px solid #000;
+  border-radius: 50%;
+  margin-right: 45px;
+}
+
+.menu__flag {
+  display: inline-block;
+  position: absolute;
+  width: 23px;
+  height: 17px;
+  cursor: pointer;
+  align-self: center;
+  right: 15px;
+  transition: transform 1s linear;
+}
+
+.menu__flag:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.new-collection__goto {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 1230px;
+  margin: 0 auto 90px auto;
+}
+
+.new-collection__goto .goto__img {
+  width: 100%;
+}
+
+.new-collection__goto .goto__title {
+  position: absolute;
+  display: inline-block;
+  color: white;
+  font-size: 40px;
+  text-transform: uppercase;
+  text-align: justify;
+  white-space: pre-line;
+  left: 63px;
+  align-self: center;
+  line-height: 1.2;
+}
+
+.goto__shop {
+  font-size: 15px;
+}
+
+.goto__arrow {
+  width: 70px;
+  cursor: pointer;
+  transition: transform 1s linear;
+}
+
+.goto__arrow:hover {
+  transform: scale(1.05, 1.05);
+}
+
+.products__title {
+  font-size: 30px;
+  margin: 0 auto 15px auto;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.products__list {
+  display: flex;
+  flex-direction: row;
+  width: 1200px;
+  height: 350px;
+  justify-content: space-between;
+  margin: 0 auto;
+  text-align: center;
+}
+
+/* rasta */
+
+.products__product-rasta {
+  align-self: flex-start;
+  text-align: left;
+  position: relative;
+  width: 280px;
+  height: 340px;
+}
+
+.products__product-rasta::after {
+  content: "";
+  border-right: 1px solid #ccc;
+  position: absolute;
+  width: 280px;
+  height: 130px;
+  top: 60px;
+}
+
+.product-rasta_g {
+  position: absolute;
+  display: inline-block;
+  right: 95px;
+  bottom: 20px;
+  background: #c4c4c4;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-rasta_g:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-rasta_b {
+  position: absolute;
+  display: inline-block;
+  right: 75px;
+  bottom: 20px;
+  background: #0e1211;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-rasta_b:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-rasta_r {
+  position: absolute;
+  display: inline-block;
+  right: 55px;
+  bottom: 20px;
+  background: #b62e3c;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-rasta_r:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.product-rasta_y {
+  position: absolute;
+  display: inline-block;
+  right: 35px;
+  bottom: 20px;
+  background: #fdb431;
+  border-radius: 50%;
+  width: 9px;
+  height: 9px;
+  transition: transform 1s linear;
+}
+
+.product-rasta_y:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/* camel */
+
+.products__product-camel {
+  align-self: flex-start;
+  text-align: left;
+  position: relative;
+  width: 280px;
+  height: 340px;
+}
+
+.products__product-camel::after {
+  content: "";
+  border-right: 1px solid #CCC;
```

Comment:
Codacy found an issue: [Expected "#CCC" to be "#ccc" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)